mllib/src/main/scala/org/apache/spark/mllib/linalg/EigenValueDecomposition.scala

```diff
@@ -32,7 +32,10 @@ private[mllib] object EigenValueDecomposition {
    *
    * @param mul a function that multiplies the symmetric matrix with a DenseVector.
    * @param n dimension of the square matrix (maximum Int.MaxValue).
-   * @param k number of leading eigenvalues required, 0 < k < n.
+   * @param k number of leading eigenvalues required,
+   *          {{{
+   *          0 < k < n
```

Comment:
- Java
  <img width="644" alt="2016-11-27 3 56 23" src="https://cloud.githubusercontent.com/assets/6477701/20642573/91f380e4-b455-11e6-8b22-751e01c2a159.png">

Scaladoc not found